app/src/main/res/values/strings.xml

```diff
@@ -26,5 +26,6 @@
     <string name="scale_pref_summary">Scale for recording text</string>
     <string name="hello_world">Hello world!</string>
     <string name="noProject">No projects found. Please choose a project in HearThis Desktop and synchronize it using the Sync button.</string>
+    <string name="no_use_without_record">This program is not much use without audio recording!</string>
```

Comment:
Since it is possible that the user accidentally denied record ability, I think this message could be made more precise and helpful. Maybe:
"Without permission to record audio. HearThis can't be used to record additional clips."